python-package/lightgbm/callback.py

```diff
@@ -131,14 +131,24 @@ def record_evaluation(eval_result: Dict[str, Dict[str, List[Any]]]) -> Callable:
 
     def _init(env: CallbackEnv) -> None:
         eval_result.clear()
-        for data_name, eval_name, _, _ in env.evaluation_result_list:
+        for item in env.evaluation_result_list:
+            if len(item) == 4:  # regular train
+                data_name, eval_name = item[:2]
+            else:  # cv
```

Comment:
What do you think about recording information about stdv as well? Just like it is done in the returned value from `cv()` function.
https://github.com/microsoft/LightGBM/blob/4aaeb22932b72aaff632b40ba596ca2533071ca2/python-package/lightgbm/engine.py#L576-L579
https://github.com/microsoft/LightGBM/blob/4aaeb22932b72aaff632b40ba596ca2533071ca2/python-package/lightgbm/engine.py#L364
https://github.com/microsoft/LightGBM/blob/ce486e5b45a6f5e67743e14765ed139ff8d532e5/tests/python_package_test/test_engine.py#L2253-L2260

Reply:
Added in 22a7bbe0a74db7121887026b8a483b08df2b0409